modules/storage-persistent-storage-pv.adoc

```diff
@@ -172,26 +168,24 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 |HostPath  | ✅ |✅ | - |  -
 |{ibm-power-server-title}  Disk | ✅ |✅  | ✅ |  ✅
 |{ibm-name} VPC Disk | ✅ |✅ | - |  -
-|iSCSI  | ✅ | ✅ |✅ |  ✅ ^[4]^
+|iSCSI  | ✅ | ✅ |✅ |  ✅ ^[3]^
 |Local volume | ✅ |✅ | - |  -
 |NFS  | ✅ | ✅ |✅ | ✅
 |OpenStack Manila  | - |✅ | - | ✅
 |{rh-storage-first}  | ✅ |✅ | - | ✅
-|VMware vSphere | ✅ |✅ | - |  ✅ ^[5]^
+|VMware vSphere | ✅ |✅ | - |  ✅ ^[4]^
 endif::[]
 |===
 [.small]
 --
 1. ReadWriteOnce (RWO) volumes cannot be mounted on multiple nodes. If a node fails, the system does not allow the attached RWO volume to be mounted on a new node because it is already assigned to the failed node. If you encounter a multi-attach error message as a result, force delete the pod on a shutdown or crashed node to avoid data loss in critical workloads, such as when dynamic persistent volumes are attached.
 
-2. ReadWriteOncePod is a Technology Preview feature.
-
-3. Use a recreate deployment strategy for pods that rely on AWS EBS.
+2. Use a recreate deployment strategy for pods that rely on AWS EBS.
```

Comment:
The style guide says re-create not recreate: https://www.ibm.com/docs/en/ibm-style?topic=word-usage (search for "re-create"). However, this is pre-existing text, so this is really out of scope; if it is possible that the UI or config uses "recreate", leave this be.